content/en/docs/instrumentation/net/getting-started.md

```diff
@@ -27,7 +27,7 @@ The following sample demonstrates manual tracing via a console app.
 
 First, install required packages:
 
-```console
+```shell
 $ dotnet add package OpenTelemetry
 $ dotnet add package OpenTelemetry.Exporter.Console
```

Comment:
This line should automatically bring in the previous package (`OpenTelemetry`), consider removing the previous line?